manim/mobject/mobject.py

```diff
@@ -1915,6 +1919,7 @@ def set_submobject_colors_by_gradient(self, *colors: Iterable[ParsableManimColor
             mob.set_color(color, family=False)
         return self
 
+    @internal
```

Comment:
why?

Reply:
Here, there was already a `set_colors_by_radial_gradient` that used this internally and returning `Self`, so I figured that should be the public one.